semgrep-core/tests/dockerfile/env-const-prop.dockerfile

```diff
@@ -0,0 +1,5 @@
+
+ENV FOO=0
+
+# MATCH:
+EXPOSE $FOO
```

Comment:
Is it guaranteed that `$FOO` here can only be the port number, or could `$FOO` be the string `22/tcp` ?

Reply:
```
ENV FOO=22/tcp
EXPOSE $FOO
```
is valid, yes.

Reply:
I am not double checking the code so I may be plain wrong, but aren't we trying to split the argument passed to `EXPOSE` into _port_ and _protocol_? Wouldn't that classify `$FOO` as _port_ when it actually contains both?

Reply:
you're right. I'm not entirely sure what to do about that. We could entirely reverse the port/protocol split, but then something like `EXPOSE $X/$Y` wouldn't work. Of course that doesn't currently work anyway...